scripts/Password_Manager/requirements.txt

```diff
@@ -0,0 +1,2 @@
+black==19.10b0
+cryptography @ file:///tmp/build/80754af9/cryptography_1616769286105/work
```

Comment:
Please check this requirements.txt !  
```
pip freeze> requirements.txt
``` 
This command added some unnecessary info to your requirements.txt.
Try running this 
```
pip list --format=freeze > requirements.txt
```

Reply:
Okey